nifi-api/src/main/java/org/apache/nifi/processor/AbstractProcessor.java

```diff
@@ -27,7 +30,12 @@ public final void onTrigger(final ProcessContext context, final ProcessSessionFa
             onTrigger(context, session);
             session.commit();
         } catch (final Throwable t) {
-            getLogger().error("{} failed to process due to {}; rolling back session", new Object[]{this, t});
+            StringWriter stacktraceWriter = new StringWriter();
```

Comment:
I would say generally no.  They are uncommon in general but when they happen they happen in bursts.  As a general rule I think we should strive to make what the user sees be something a lot more friendly than a wild stack trace.  But, the logs should have them.  We did discuss this on dev list a while back and there were some great inputs from various folks.  We def need to do something here.